### PR TITLE
remove unnecessarily duplication since types collapsed

### PR DESCRIPTION
**Pull request by deads2k**
We collapsed duplicate types into client-go, so we get to clean this up.

@sttts as promised.

### CODE REVIEW

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -413,27 +417,6 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 			}
 		}
 
-		// client-go and kuberenetes generated clients are incompatible because the runtime
```
Comment (sttts) [inline cmd/kubelet/app/server.go]: :+1:

File: cmd/kubelet/app/server.go [hunk 2]
```diff
@@ -396,6 +396,10 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 			if err != nil {
 				glog.Warningf("New kubeClient from clientConfig error: %v", err)
 			}
+			externalKubeClient, err = clientgoclientset.NewForConfig(clientConfig)
+			if err != nil {
+				glog.Warningf("New kubeClient from clientConfig error: %v", err)
```
Comment (sttts) [inline cmd/kubelet/app/server.go]: It's not really something the admin will understand.

**Comment by sttts** [comment]
Looks good. Only the warning text is not good.

File: cmd/kubelet/app/server.go (hunk 2 above)
Comment (deads2k) [inline cmd/kubelet/app/server.go]: > It's not really something the admin will understand.

Prexisting from the original

**Comment by deads2k** [comment]
approved based on https://github.com/kubernetes/kubernetes/issues/40363

**Pull request merged**